.travis.yml

```
@@ -154,8 +153,42 @@ jobs:
       after_deploy:
         - npm run doc-cloudfront
 
+    - stage: Deploy latest release on NPM
+      if: type = push AND branch =~ /^master$/
```

Avoid regexp when they're not needed :wink: 
```suggestion
      if: type = push AND branch = master
```